data-source/json/test_schema_theme_census.json

```diff
@@ -0,0 +1,68 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "0",
+    "title": "A questionnaire to check schema census theme.",
+    "theme": "census",
+    "description": "A questionnaire to check schema census theme.",
+    "messages": {
+        "NUMBER_TOO_LARGE": "Number is too large",
+        "NUMBER_TOO_SMALL": "Number cannot be less than zero",
+        "INVALID_NUMBER": "Please enter an integer"
+    },
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
```

Comment:
No metadata should be required, but it looks like launcher doesn't work when there is none. We can removed `period_id` and `ru_name` here though.

Reply:
Fixed, thanks!